bin/docker-compose

```diff
@@ -178,6 +178,10 @@ function set_nginx_vars() {
   if [[ -n ${NGINX_CONFIG_PATH-} ]]; then
     NGINX_CONFIG_PATH=$(cd "$TOOLKIT_ROOT"; realpath "$NGINX_CONFIG_PATH")
   fi
+  OVERLEAF_TRUSTED_PROXY_IPS=$(read_variable OVERLEAF_TRUSTED_PROXY_IPS || echo "")
+  if [[ "${OVERLEAF_TRUSTED_PROXY_IPS:-null}" == "null" ]]; then
+    OVERLEAF_TRUSTED_PROXY_IPS="loopback,172.16.0.0/12"
```

Comment:
Suggest adding a comment that 172.16.0.0/12 is the default private ip range used by docker.

Reply:
:+1: I've also updated the changelog, including a note for customers with external TLS proxies.